genesyscloud/responsemanagement_response/resource_genesyscloud_responsemanagement_response_utils.go

```diff
@@ -54,6 +54,7 @@ func buildResponseTexts(responseTexts *schema.Set) *[]platformclientv2.Responset
 
 		resourcedata.BuildSDKStringValueIfNotNil(&sdkResponseText.Content, responseTextMap, "content")
 		resourcedata.BuildSDKStringValueIfNotNil(&sdkResponseText.ContentType, responseTextMap, "content_type")
+		resourcedata.BuildSDKStringValueIfNotNil(&sdkResponseText.VarType, responseTextMap, "type")
```

Comment:
is this "Vartype" or "Type" ?

Reply:
No, it is "type" only. I test, verified and confirmed.